ucte/ucte-converter/src/main/java/com/powsybl/ucte/converter/UcteExporter.java

```diff
@@ -350,21 +343,6 @@ private static void convertXNode(UcteNetwork ucteNetwork, DanglingLine danglingL
         }
     }
 
-    /**
-     * Create a {@link UcteNode} object from a MergedXnode and add it to the {@link UcteNetwork}.
-     *
-     * @param ucteNetwork The target network in ucte
-     * @param mergedXnode The MergedXnode extension used to create the XNode
-     * @param context The context used to store temporary data during the conversion
-     */
-    private static void convertXNode(UcteNetwork ucteNetwork, MergedXnode mergedXnode, UcteExporterContext context) {
-        UcteNodeCode xnodeCode = context.getNamingStrategy().getUcteNodeCode(mergedXnode.getCode());
-        String geographicalName = ((Identifiable<?>) mergedXnode.getExtendable()).getProperty(GEOGRAPHICAL_NAME_PROPERTY_KEY, "");
```

Comment:
You are removing something that is not supported by the refactoring: the support of the geographical name and the support of the Xnode status.